documentation/src/docs/security/pravega-security-encryption.md

```diff
@@ -0,0 +1,34 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega Encryption
+## Encryption of data in flight 
+Pravega ensures that all the data in flight can be passed on in an encrytped manner.
+The different channels [channels](https://github.com/pravega/pravega/wiki/PDP-23:-Pravega-security----encryption-and-Role-Based-Access-Control#b-encryption-of-data-in-flight-over-network-and-in-tier-1)
+can be configured with TLS and encryption enabled for them.
+### Certificate management
+Pravega expects administrator user to create and manage certificate creation, deployment and management. 
+Pravega provides different configuration parameters where certificates for different communication channels can be specified.
+### Encrypted data flow between Pravega client and Pravega controller and segmentstore
+Pravega uses same cert to interact with both controller and segmentstore. The certificates need to be mentioned specifically on the client and the server.
+These certificates are not loaded from the trust store.
+### Encrypted data flow between Pravega and tier-1 (Apache Bookkeeper)
+Apache bookkeeper supports JKS based truststore. Segmentstore uses JKS based truststore to interact with it. The configurations
+#### Segmentstore and bookkeeper
+Pravega Segmentstore uses Apache bookkeeper as Tier-1 storage. 
```

Comment:
`Tier-x` is not being written consistently, please use `Tier 1` and `Tier 2` when referring to Pravega storage tiers, and `tier` small font when talking about tiers in general.